test/evergreen.yml

```diff
@@ -255,6 +278,11 @@ tasks:
         vars:
           posix_configure_flags: --enable-silent-rules --enable-diagnostic --enable-strict --enable-python
 
+  - name: compile-sanitizer
```

Comment:
It looks there's an existing task `compile-asan` having the same configure flags, which might be reused with an overload of `posix_configure_flags` to meet the need here.

Reply:
Ah we went circles and Ravi and I have decided not to use task for compilation. We will follow the same style for rest of the tickets as well.